launcher/src/main/java/org/apache/spark/launcher/ChildProcAppHandle.java

```diff
@@ -103,67 +46,22 @@ public synchronized void kill() {
       try {
         childProc.exitValue();
       } catch (IllegalThreadStateException e) {
-        // Child is still alive. Try to use Java 8's "destroyForcibly()" if available,
```

Comment:
Why is this being removed? This is not supposed to be a "clean kill", and `destroyForcibly` works in certain cases where the child JVM is deadlocked and doesn't respond to `destroy()` (which just sends a SIGTERM).